config/environments/production.rb

```diff
@@ -44,17 +44,19 @@
   # Force all access to the app over SSL, use Strict-Transport-Security, and use secure cookies.
   config.force_ssl = true
 
+  # Skip http-to-https redirect for the default health check endpoint.
+  # config.ssl_options = { redirect: { exclude: ->(request) { request.path == "/up" } } }
+
```

Comment:
It would be good to test this PR in the dev environment, I'm a bit confused about why we don't need this line to be uncommented...

Reply:
Because it says to _not_ redirect `/up` from http to https?

This line is new and added by the update script, and commented out by default.

Reply:
Yeah I'm just wondering if we do actually need `/up` to be http (not https), as I thought the healthcheck happened behind the SSL termination 🤔

Reply:
ill check with Andy

Reply:
Yes, I believe we need `/up` to accept HTTP so the load balancer can health check it.

Reply:
I wonder if the `config.assume_ssl` option is somehow making the other ssl options irrelevant...

Reply:
ARGH

Reply:
@lfdebrux what do you mean appears to be working? can you point me at it pls? can it be that it's just following the redirect?

Reply:
What do you think we should do here @lfdebrux

Reply:
Let's keep the Rails default (commented out) and just keep an eye on it. At some point I'll have a poke around and see if I can figure out what's going on, but it's mainly just curiosity that is motivating me...